### PR TITLE
separate FileRequest from FileResponse

### PR DESCRIPTION
**Pull request by felipeska**
Hello @go-cristian some changes here:

- Delete `request` property from `FileResponse`.
- Rename `response` property from `FileResponse` to `body`.
- New structure called `FileRecorded`.

### CODE REVIEW

File: test-record-interceptor/src/main/java/dev/cgomez/testrecordinterceptor/FileRecorded.kt [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package dev.cgomez.testrecordinterceptor
+
+import okhttp3.Protocol
+import okhttp3.Request
+import okhttp3.RequestBody.Companion.toRequestBody
+import okhttp3.Response
+import okhttp3.ResponseBody.Companion.toResponseBody
+
+data class FileRecorded(
+  val responseRecorded: FileResponse,
```
Comment (go-cristian) [inline test-record-interceptor/src/main/java/dev/cgomez/testrecordinterceptor/FileRecorded.kt]: I like this naming convention, can we use it for the typings too?
maybe something like `RecordedResponse` & `RecordedRequest` is more accurate.

**Review by go-cristian — CHANGES_REQUESTED** [review]
Thanks, @felipeska for this restructure, I think changing the naming is valuable here. I would ask for this change and for sure this will be merged.

File: test-record-interceptor/src/main/java/dev/cgomez/testrecordinterceptor/FileRecorded.kt (hunk 1 above)
Comment (felipeska) [reply test-record-interceptor/src/main/java/dev/cgomez/testrecordinterceptor/FileRecorded.kt]: absolutely! I changed the file names to be more accurate.

**Review by go-cristian — APPROVED** [review]
Thanks for it... I need to add some docs and publish this.

**Pull request merged**